### DISCUSSION TITLE
Is it possible to merge one website data to another

### DISCUSSION THREAD

**Discussion by woss** [plausible/analytics#2862] [Anything Else]
I have this issue when i renamed/rebranded the website and now I would like to merge all the data from the old one to the new one. Is this possible?

**Comment by metmarkosaric** [comment]
hi @woss! it's not possible to merge data from two different dashboards but you can change the domain name of your site directly in your Plausible site settings. see https://plausible.io/docs/change-domain-name

**Reply by woss** [reply]
Pozz,

I have self hosted version and it seems I cannot edit it.
